packages/components/inputs/styles.js

```diff
@@ -29,20 +28,26 @@ const getInputStyles = (props, theme) => {
     padding: 0 ${overwrittenVars.spacingS};
     transition: border-color ${overwrittenVars.transitionStandard},
       background-color ${overwrittenVars.transitionStandard},
-      color ${overwrittenVars.transitionStandard};
+      color ${overwrittenVars.transitionStandard},
+      box-shadow ${overwrittenVars.transitionStandard};
     width: 100%;
 
     &::placeholder {
       color: ${overwrittenVars[designTokens.placeholderFontColorForInput]};
     }
     &:active,
-    &:focus {
+    &:focus,
+    &:hover {
       border-color: ${overwrittenVars[
         designTokens.borderColorForInputWhenFocused
       ]};
     }
+    &:focus {
+      box-shadow: inset 0 0 0 2px
```

Comment:
Focusing adds a thicker border.

We're using a box-shadow styled as a border because using a normal border would push the contents inward in our current input layout. The alternative would be reconstructing the layout to use absolute positioning.

Reply:
Thanks for the explanation!